osp/utils/zulip_api.py

```diff
@@ -23,7 +42,16 @@ def get_messages(zulip_id):
     return result
 
 
-def get_newest_message(zulip_id):
+def get_newest_message(zulip_id: int) -> dict:
```

Comment:
@isabelcosta This function is used for getting the newest message timestamp in `zulip_stat.py`. So I think it should return only timestamp. Should I create a new issue for this?

Reply:
yap create an isolated issue, so we keep PRs easy to review ;)